### ISSUE TITLE
Anti-Complement factor H antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7062]
**Preferred term label:**
Anti-Complement factor H antibody positivity

**Synonyms**
CFH autoantibodies

**Definition (free text, please give PubMed ID)**
PMID: 34032207, PMID: 34032209

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:329918, ORPHA:93571, ORPHA:97560, ORPHA:93552, ORPHA:93552

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
